src/environment/stella_environment.cpp

```diff
@@ -130,6 +136,14 @@ reward_t StellaEnvironment::act(Action player_a_action, Action player_b_action)
     sum_rewards += oneStepAct(player_a_action, player_b_action);
   }
 
+  if (!m_record_screen_dir.empty()) {
+    static int recorded_frame = 0;
+    std::stringstream ss;
+    ss << m_record_screen_dir << "/" << setfill('0') << setw(6) <<
+        std::to_string(recorded_frame++) << ".png";
```

Comment:
Is there a reason to use std::to_string rather than pass recorded_frame to the stringstream? std::to_string is new in C++11.